qa/tasks/cephfs/filesystem.py

```diff
@@ -263,18 +263,11 @@ def json_asok(self, command, service_type, service_id, timeout=None):
             log.debug("_json_asok output empty")
             return None
 
-    def is_addr_blocklisted(self, addr=None):
-        if addr is None:
-            log.warn("Couldn't get the client address, so the blocklisted "
-                     "status undetermined")
-            return False
```

Comment:
Perhaps default value of addr was set deliberately so that test doesn't crash when get_global_addr() return None?

```
qa/tasks/cephfs/test_misc.py:187:            self.assertTrue(self.mds_cluster.is_addr_blocklisted(
qa/tasks/cephfs/test_misc.py-188-                self.mount_a.get_global_addr()))
--
qa/tasks/cephfs/test_sessionmap.py:223:        self.wait_until_true(lambda: self.mds_cluster.is_addr_blocklisted(
qa/tasks/cephfs/test_sessionmap.py-224-            self.mount_a.get_global_addr()), timeout=30)
--
```

get_global_addr() [returns self.addr](https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/fuse_mount.py#L505) the default value of which is [set to None](https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/fuse_mount.py#L33).

Reply:
Yeah, I see those possibilities of `get_global_addr` returning `None`. But still the current code change still works as expected (`None` in blocklist returns `False`). So the only benefit I see is logging. The `log.warn("Couldn't get the client address....` would log if addr is `None` somehow. That might be the reason it was kept by the original author.

Considering these^ things it would be a better idea to log if the address not found? It covers both scenarios.